### ISSUE TITLE
"genes":null for guideline queries 

### ISSUE THREAD

**Issue by mumtahena** [cpicpgx/cpic-data#8]
### Problem: __"genes"__ for the guideline queries are resulting as __'null'__ for all but the PPI and NSAID guidelines.

curl --location --request GET 'https://api.cpicpgx.org/v1/guideline?select=*,publication(title,pmid,year)&order=genes'

[{"id":110076,"version":1,"name":"CYP2C19 and Proton Pump Inhibitors","url":"https://cpicpgx.org/guidelines/cpic-guideline-for-proton-pump-inhibitors-and-cyp2c19/","pharmgkbid":["PA166219103","PA166219301"],__"genes":["CYP2C19"]__,"publication":[]},
{"id":110058,"version":2,"name":"CYP2C9 and NSAIDs","url":"https://cpicpgx.org/guidelines/cpic-guideline-for-nsaids-based-on-cyp2c9-genotype/","pharmgkbid":["PA166191841"],__"genes":["CYP2C9"]__,"publication":[]},
{"id":100416,"version":4,"name":"CYP2D6 and Codeine","url":"https://cpicpgx.org/guidelines/guideline-for-codeine-and-cyp2d6/","pharmgkbid":["PA166104996"],__"genes":null__,"publication":[{"title":"Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for Codeine Therapy in the Context of Cytochrome P450 2D6 (CYP2D6) Genotype","pmid":"22205192","year":2011},
{"title":"Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines for cytochrome P450 2D6 (CYP2D6) genotype and codeine therapy: 2014 Update","pmid":"24458010","year":2014}]},
{"id":100421,"version":4,"name":"HLA-B and Abacavir","url":"https://cpicpgx.org/guidelines/guideline-for-abacavir-and-hla-b/","pharmgkbid":["PA166104997"],__"genes":null__,"publication":[{"title":"Clinical Pharmacogenetics Implementation Consortium Guidelines for HLA-B Genotype and Abacavir Dosing","pmid":"22378157","year":2012},
and so on.

**Comment by whaleyr** [comment]
You're right, the `genes` property of the `guidelines` model has not been fully populated yet. I'll get that filled in soon.

**Comment by whaleyr** [comment]
I pushed a fix for this so you should no longer be seeing missing gene information in the guideline table/endpoint. Closing the issue for now, but if you still see a problem related to this feel free to re-open or comment.
